### DISCUSSION TITLE
이번 주 참석이 어려울 것 같습니다😢

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#205] [General]
개인사정으로 이번주 참가가 어려울 것 같습니다ㅠ
늦게 알려드려 죄송합니다ㅜ;

**Comment by jongfeel** [comment]
그러셨군요. 알겠습니다.

다음은 3장, 4장이고 분량이 많아서 준비를 잘 해주시고
꼭 뵜으면 좋겠습니다.
